lib/Analysis/Utility.cpp

```diff
@@ -392,14 +392,42 @@ static bool supportMFMAGranularity(int m, int n, int k) {
   return false;
```

Comment:
Is this `supportMFMAGranularity()` a duplicate of the one in sematic.py?

Reply:
Yes.
Unfortunately, this is a design flaw.
I've created an issue somehow related to this: https://github.com/ROCmSoftwarePlatform/frameworks-internal/issues/6172